bridges/otelzap/config.go

```diff
@@ -0,0 +1,82 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otelzap // import "otelzap"
```

Comment:
Have you added ` // import "otelzap"` manually? The import path should be `go.opentelemetry.io/contrib/bridges/otelzap` and should be generated automatically by `make`.